src/computed_field/computed_field_subobject_group.cpp

```diff
@@ -621,6 +621,36 @@ int Computed_field_element_group::removeElementFaces(cmzn_element_id parent)
 	return return_code;
 }
 
+void Computed_field_element_group::destroyedAllElements()
+{
+	this->labelsGroup->clear();
+	this->invalidateIterators();
+	change_detail.changeRemove();
+	this->field->setChangedPrivate(MANAGER_CHANGE_PARTIAL_RESULT(Computed_field));
```

Comment:
I would be very tempted to put this code inside its own method because it is repeatedly used.